docs/contribute-module.md

```diff
@@ -13,12 +13,13 @@ Follow these steps to create or update a module.
 ## Before you begin
 
 - If you're running Microsoft Windows, set up Windows Subsystem for Linux [WSL](https://ubuntu.com/wsl) and run commands within WSL.
-- Make sure that you have a current version of Python 3 and pip.
+- Make sure that you have a current version of Python 3 and pip. 
+  - _Notes for the macOS users:_ The out-of-the-box Python (Python 2.x) which comes with macOS is not suitable. The installation instructions for the latest Python 3.x version on macOS can be found [here](python3-mac-installation.md).
```

Comment:
Do we need to tell people this? Have we run into issues? Python 2 has been deprecated for years. 

If you still think a notice is required, let's just tell them how to check.

```suggestion

    To check your installed version, run `python --version`.
```